virt/virt-2-5-release-notes.adoc

```diff
@@ -70,8 +70,11 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //CNV-6824 - Offline virtual machine snapshots
 
 //CNV-6825 - Smart cloning with snapshots
-
 //CNV-4089 - Cloning a disk quickly using the storage array
+// These two release note stories are combined in this single release note.
+* Clone disks efficiently using smart-cloning, providing that your storage provider
```

Comment:
Reworded to avoid "providing/provided":

_* Clone virtual disks efficiently and quickly using smart-cloning. Smart-cloning
occurs automatically when you create a DataVolume with a PersistentVolumeClaim (PVC) source. 
Your storage provider must support the CSI Snapshots API to use smart-cloning._